plutus-metatheory/src/html/Agda.css

```diff
@@ -0,0 +1,39 @@
+/* Aspects. */
```

Comment:
duplicate file

Reply:
I recall that that was needed for now as we have two kinds files that look for css in two different places (`blah.lagda` -> `blah.html` via agda and `blah.lagda.md` -> `blah.md` -> `blah.html` via agda and then jekyll). I can try to fix it.